modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for {hcp-title}
+
+
+Description needed !!!!
+
+.Procedure
+
+. Describe {hcp-title} clusters with image registry:
++
+[source,terminal]
+----
+$ rosa describe cluster -c <cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       rosa-hcp-test
+Domain Prefix:              rosa-hcp-test
+Display Name:               rosa-hcp-test
+ID:                         <cluster_id>
+External ID:                <cluster_hcp-id>
+Control Plane:              ROSA Service Hosted
+OpenShift Version:          4.17.0
+Channel Group:              stable
+DNS:                        <dns>
+AWS Account:                <AWS_id>
+AWS Billing Account:        <AWS_id>
```

Comment:
```suggestion
AWS Account:                <aws_id>
AWS Billing Account:        <aws_id>
```
I won't keep commenting on these, since I don't know if the module is staying or going, but if it's staying, then lowercase all of these.